src/manage_cyhy_ops/manageoperators.py

```diff
@@ -36,94 +36,82 @@ def _get_cyhy_ops_list(self):
+                'The CyHy Operators parameter "%s" does not exist in region "%s".',
+                self.cyhy_ops_key,
+                self.region,
             )
-        except ClientError as e:
-            logging.error(e)
+        except ClientError as err:
+            logging.error(err)
 
         return users
 
     def _update_cyhy_ops_users(self, username: str, remove: bool = False) -> int:
         """Update the list of CyHy Operators to use when an instance is built."""
         users: List[str] = self._get_cyhy_ops_list()
+        update_msg: str = "performed no operations on"
 
-        logging.debug("Current CyHy Operators: {users}.")
+        logging.debug("Current CyHy Operators: %s.", users)
 
         if remove:
             if username not in users:
                 logging.warning(
-                    f'User "{username}" is not in the list of active '
-                    f'CyHy Operators in region "{self.region}".'
+                    'User "%s" is not in the list of active CyHy Operators in region "%s".',
+                    username,
+                    self.region,
                 )
             else:
                 users.remove(username)
-                update_msg = f'removed "{username}" from'
+                update_msg = 'removed "%s" from'
         else:
             if username in users:
                 logging.warning(
-                    f'User "{username}" is already in the list of active '
-                    f'CyHy Operators in region "{self.region}".'
+                    'User "%s" is already in the list of active CyHy Operators in region "%s".',
+                    username,
+                    self.region,
                 )
             else:
                 users.append(username)
-                update_msg = f'added "{username}" to'
+                update_msg = 'added "%s" to'
 
-        if len(users) == 0:
-            try:
-                logging.warning(
-                    "No CyHy Operators left, deleting CyHy Operators parameter "
-                    f'from region "{self.region}".'
-                )
-                # Response is an empty dictionary on success.
-                self._client.delete_parameter(Name=self.cyhy_ops_key)
-            except ClientError as e:
-                logging.error(
-                    "Unable to delete the CyHy Operators parameter in "
-                    f'region "{self.region}".'
-                )
-                logging.error(e)
-                return 1
-        else:
-            updated_users = ",".join(users)
+        updated_users = ",".join(sorted(users))
 
-            logging.debug(f'New CyHy Operators value: "{updated_users}".')
+        logging.debug('New CyHy Operators value: "%s".', updated_users)
+
+        try:
+            # The SSM response on success currently only contains a version
+            # number and the parameter tier.
+            # Neither are useful to us at this time, so we don't store them..
```

Comment:
Is the double-period intentional here?  I saw another one elsewhere.

```suggestion
            # Neither are useful to us at this time, so we don't store them.
```

Reply:
Nope, nice catch. Fixed both instances in 5b1711c.